.env.example

```diff
@@ -7,3 +7,4 @@ IDAPI_BASE_URL=
 PLAY_SESSION_COOKIE_SECRET=
 DEFAULT_RETURN_URI=
 SIGN_IN_PAGE_URL=
+IS_HTTPS=false
```

Comment:
think in the example it should be left blank as you can use `localhost` non https, or `thegulocal` with https

you'll also want to update the dev config .env files in the s3 bucket to reflect these new parameters if you haven't done so already

Reply:
done